### PR TITLE
feat: [DHIS2-13661] program stage working list default filters

### PR DESCRIPTION
**Pull request by simonadomnisoru**

### CODE REVIEW

**Comment by jasminenguyennn** [comment]
Hi @simonadomnisoru, great work in this PR 🎉
I only have few comments
- When program stage is selected, both Report data and Event status stay disabled 
Screenshot 2022-09-12 at 09 45 19
- nitpick:  the label should be consistent `Event status` instead of `Event Status`

File: src/core_modules/capture-core-utils/featuresSupport/support.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// @flow
+export const FEATURES = {
```
Comment (jasminenguyennn) [inline src/core_modules/capture-core-utils/featuresSupport/support.js:2]: I think we should freeze the object

**Comment by simonadomnisoru** [comment]
> Hi @simonadomnisoru, great work in this PR 🎉 I only have few comments
> 
> * When program stage is selected, both Report data and Event status stay disabled
> 
> Screenshot 2022-09-12 at 09 45 19
> 
> - nitpick: the label should be consistent `Event status` instead of `Event Status`

Hey @jasminenguyennn, 

Thanks for the review! I implemented the small suggestions. 
Regarding the comment about what happens when a program stage is selected, that part of the feature has not been implemented yet. I am currently working to analyze it [DHIS2-13662](https://dhis2.atlassian.net/browse/DHIS2-13662) and I will create more tickets and PRs in the next few days. 
Thanks!

**Review by jasminenguyennn — APPROVED** [review]
👍  LGTM

**Pull request merged**